src/constants.tsx

```diff
@@ -3,8 +3,8 @@ export const devnetUrl =
 
 export const networks = {
   local: "http://localhost:8080",
-  devnet: devnetUrl,
-  test: "https://rosetta.aptosdev.com/",
+  Devnet: devnetUrl,
+  Testnet: "https://rosetta.aptosdev.com/",
```

Comment:
@CapCap what is `rosetta` used for? On Petra, `Testnet` points to the future ait3 network.

Reply:
I believe it's a partner test network. Coinbase?